ddio/sdljoy.cpp

```diff
@@ -318,19 +318,22 @@ void joy_GetPos(tJoystick joy, tJoyPos *pos) {
 static int joyGetNumDevs(void) {
   int found = 0;
 
+  int joyCount = 0;
+  SDL_GetJoysticks(&joyCount);
```

Comment:
We might have a little mem leak here. [SDL_GetJoysticks](https://wiki.libsdl.org/SDL3/SDL_GetJoysticks) states that the returned array "should be freed with [SDL_free](https://wiki.libsdl.org/SDL3/SDL_free)() when it is no longer needed."

Reply:
indeed, it should be fixed now!